spec/views/event/application_market/_participation.html.haml_spec.rb

```diff
@@ -0,0 +1,15 @@
+#  Copyright (c) 2025, Hitobito AG. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+require "spec_helper"
+
+describe "event/application_market/_participation.html.haml" do
+  let(:event) { EventDecorator.decorate(Fabricate(:course, groups: [groups(:top_layer)])) }
+  let(:participation) { Fabricate(:event_participation, event: event) }
+
+  it "participation without application isn't causing an internal server error" do
+    expect { render locals: {p: participation.decorate} }.not_to raise_error
```

Comment:
Also for this, I would rather render the view and check the content (when content exists how you want it, there is no error). Add a check for the expected text like person_location_information or whatever else is rendered in the view

Reply:
But this isn't the bug is it? The Bug is that we get a 500 Internal Server error so the best spec to check this is to verify if the rendering of the page causes any error. The check if the right content is displayed has nothing to do with the Bug I fixed.